src/libImaging/Histo.c

```diff
@@ -29,10 +29,12 @@
 void
 ImagingHistogramDelete(ImagingHistogram h)
 {
-    if (h->histogram) {
-        free(h->histogram);
+    if (h) {
+        if (h->histogram) {
+            free(h->histogram);
+        }
+        free(h);
```

Comment:
I suppose this change is just to help in theory - but I'll still point out that in practice, with the other changes from this PR, `ImagingHistogramDelete` never receives a NULL argument.